apps/asap-server/src/utils/sync-google-calendar.ts

```diff
@@ -1,21 +1,22 @@
 import { google, calendar_v3 as calendarV3, Auth } from 'googleapis';
+import { DateTime } from 'luxon';
 import logger from './logger';
 
 export type SyncCalendarFactory = (
   syncToken: string | undefined,
-  syncEvent: (
-    event: calendarV3.Schema$Event,
-    defaultCalendarTimezone: string,
-  ) => Promise<void>,
+  syncEvent: SyncEvent,
   auth: Auth.GoogleAuth | Auth.OAuth2Client,
 ) => (googleCalendarId: string) => Promise<string | undefined | null>;
 
+interface SyncEvent {
+  (event: calendarV3.Schema$Event, defaultCalendarTimezone: string): Promise<
+    unknown
```

Comment:
probably best to return `Promise<void>` if you don't really expect it to return anything, otherwise it's a bit misleading

Looking at 
https://github.com/yldio/asap-hub/pull/691/files#diff-316c33d3a09eac72218fdebd1ac2dc43ec0d0667a3fbd4fb872fabbdc7e919b9R64
it appears that the signature should actually be a little different and the return type should contain something?

Reply:
I could make this a RestEvent but I had plans to make this generic and extract this piece of code. 🤷🏻‍♂️

Reply:
well yeah you could just give it a return type that is not directly linked to our data store, something generic.